src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+# Setting up Printers
+
+Most printers are plug-and-play in Pop!\_OS. Add a network or USB connected printers in `Settings` ➞ `Printers`. Clicking `Add a Printer` will automatically search for any connected printers. Printers can also be added manually by clicking `Additional Printer Settings`.
+
+![Printer Settings Screen](/images/printers/printer-settings.png)
+
+---
+
+## CUPS and Printer Compatibility
+
+Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, print queues, and recognizes connected printers. While CUPS recognizes a wide range of printer models, printer functionality may vary. Check if your printer is compatible using [OpenPrinter.com](https://www.openprinting.org/printers).
+
+## Checking Printer Status in CUPS
+
+Open this web page to look at the CUPS (Common Unix Printing System) configuration and status window:
+
+[localhost:631](http://localhost:631)
+
+The status window will show current print jobs, detected printers, and other information about the printing system. If you would like to share this printer with others on your local network, click on the 'Admin' link, under Server, click on the "Share printers connected to this system" and save the changes. Other computers on your network should than see that printer. When there is a prompt for your username and password, use your user name, and password used to login.
```

Comment:
> click on the 'Admin' link, under Server, click on the "Share printers connected to this system" and save the changes.

Is `Admin` supposed to be `Administration`?

This seems like a run-on sentence. If you want to use commas within list items, then the list needs to be delimited with something else, like semicolons or bullet points:

> If you would like to share this printer with others on your local network, click on the 'Administration' link; under Server, click on the "Share printers connected to this system"; and save the changes.

> When there is a prompt for your username and password, use your user name, and password used to login.

It seems extremely arbitrary that you're calling out the difference between a `username` and a `user name`. The last comma is grammatically unnecessary and incorrect. I would also reword to avoid "use" being present twice so close together:

> When there is a prompt for your username and password, enter the username and password you use to log in to the computer.

This instruction will only work if the user is part of the `lpadmin` group. This can be checked using the `groups` command on the terminal. In Pop!_OS, an admin user will usually be part of this group (while a standard user will not be part of this group), although there are cases where an admin isn't part of the group (especially in previous versions that might have been upgraded.)

Reply:
This was another section pulled from a support article, but that doesn't excuse the fact that I should have scrutinized the content before adding it. Thanks for breaking this down, I'll add your suggested changes.

Reply:
This paragraph should also be its own multi-step procedure. I'm moving this CUPS section after the "Adding a Printer" section since it deals with advanced configuration that would only be performed after adding a printer.